src/web/models.py

```diff
@@ -63,6 +63,7 @@ class Service(models.Model):
     duration = models.IntegerField(default=0)
     time_before = models.IntegerField(default=0)
     time_after = models.IntegerField(default=0)
+    payout = models.IntegerField(default=50, null=True)
```

Comment:
Maybe make this `default=0` so that later you don't try to do: `5 + None + 10` giving your value errors